<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Euler</class>
 <widget class="QMainWindow" name="Euler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Euler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Initial point (Xi)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Final point (Xf)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Increment costant/Slope of fuction (h)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="Xi_spinbox"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Xf_spinbox"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="h_spinbox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="METHOD_BOTTOM">
        <property name="text">
         <string>METHOD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ERASE_BOTTON">
        <property name="text">
         <string>ERASE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>CLOSE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOrigial">
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="actionIntegrated">
   <property name="text">
    <string>Integrated</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="actionUse_the_Euler_Method">
   <property name="text">
    <string>Use the Euler Method</string>
   </property>
  </action>
  <action name="actionErase_current_data">
   <property name="text">
    <string>Erase current data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
